### ISSUE TITLE
add reference to Protective Load Balance

### ISSUE THREAD

**Issue by dwcarder** [dwcarder/draft-cc-v6ops-wlcg-flow-label-marking#28]
https://dl.acm.org/doi/pdf/10.1145/3544216.3544226
https://www.phoronix.com/news/TCP-PLB-Linux
https://github.com/google/bbr/blob/a23c4bb59e0c5a505fc0f5cc84c4d095a64ed361/net/ipv4/tcp_bbr2.c#L2257
https://github.com/google/bbr/blob/6e321d1c986a88e21dcbb46005668cd874de01da/net/ipv4/tcp_bbr.c#L1118C13-L1118C20

We’d need WLCG nodes to not use the feature.
An “other clashing uses of flow label” inventory may be handy, this could be combined with the existing paragraph about
RFC7098.

**Comment by timchown** [comment]
Yeah, if our draft is discussing compliance to standards (other RFCs) another section on other known uses would be useful, so we can see whether the clashes might pose an issue, and how to avoid them.

PLB is one.  I’ll email Lorenzo and copy you.  There may be an option to turn PLB off in BBR.
